### ISSUE TITLE
Architecture detection for 'lastversion install' is busted

### ISSUE THREAD

**Issue by cobalt2727** [dvershinin/lastversion#86]
Tried to install [Webcord](https://github.com/SpacingBat3/WebCord/releases/latest) on an x86_64 system and the following happened:

```bash
 cobalt@fedora-latitude-5175
    os ~ fedora
    sh ~ bash
    up ~ 00:03
   cpu ~ Intel m5-6Y57
   mem ~ 7780MB
  host ~ Latitude 5175
  kern ~ 6.0.10
  term ~ tmux: server
  ▅▅▅▅▅▅▅▅▅▅▅▅

[cobalt@fedora-latitude-5175 ~]$ sudo lastversion install webcord

Last metadata expiration check: 0:05:10 ago on Thu 01 Dec 2022 03:38:38 AM EST.
webcord-3.10.0-1.armv7hl.rpm                        7.5 MB/s |  25 MB     00:03    
webcord-3.10.0-1.x86_64.rpm                          12 MB/s |  82 MB     00:06    
Error: 
 Problem: conflicting requests
  - package webcord-3.10.0-1.armv7hl does not have a compatible architecture
(try to add '--skip-broken' to skip uninstallable packages)
```

**Comment by dvershinin** [comment]
@cobalt2727 should be fixed in the latest `lastversion` v2.4.8. Please give it a try.

**Comment by cobalt2727** [comment]
I completely forgot to come back and report this as fixed, but better late than never...
